### PR TITLE
Use proper names for Calico certificates and remove unnecessary ceritifcates

### PR DESCRIPTION
**Pull request by r7vme**
Part of: https://github.com/giantswarm/giantswarm/issues/4173

Naming aligned with Calico CNI script https://github.com/projectcalico/cni-plugin/blob/release-v3.2/k8s-install/scripts/install-cni.sh#L40-L55

### CODE REVIEW

**Review by xh3b4sd — APPROVED** [review]
When you have the migration in mind then lets go with this.

File: files.go [hunk 1]
```diff
@@ -72,19 +72,6 @@ func newFilesClusterCommon(cluster Cluster) Files {
...
-			AbsolutePath: "/etc/kubernetes/ssl/etcd/calico-client-ca.pem",
-			Data:         cluster.CalicoEtcdClient.CA,
-		},
-		{
-			AbsolutePath: "/etc/kubernetes/ssl/etcd/calico-client-crt.pem",
-			Data:         cluster.CalicoEtcdClient.Crt,
-		},
-		{
-			AbsolutePath: "/etc/kubernetes/ssl/etcd/calico-client-key.pem",
-			Data:         cluster.CalicoEtcdClient.Key,
```
Comment (kopiczko) [inline files.go]: Can we keep `CalicoEtcdClient` and remove `CalicoClient` instead? I think it's clearer to have `Etcd` in the name.

**Comment by r7vme** [comment]
PTAL, addressed comment.

Migration is easy because we don't use Calico certificates now. Anyway all tracked here https://github.com/giantswarm/giantswarm/issues/4173

**Review by kopiczko — APPROVED** [review]

**Review by tuommaki — APPROVED** [review]
LGTM, but might be worth to properly test (as last time I modified this, there were issues later that hit the fan).

**Comment by r7vme** [comment]
> Can we just simply remove the certs there?

Yes.

So the calico certs are no longer needed?

We had two copies of calico certs and none were used. This change just removes one and changes to proper filenames for another.

So in short, they are not used, so yes can be removed.

> What about the certs being there? Will they be cleaned up somehow?

Seems not. Do you have an idea how it can be done?

**Comment by xh3b4sd** [comment]
> Seems not. Do you have an idea how it can be done?

This is part of a migration plan we should have. We could just add some goroutine to `cluster-operator` checking if these things are there and if so delete them. Quick and dirty. Deploy. Wait. Cleanup code.

**Comment by xh3b4sd** [comment]
I remove the calico certs from `kubernetesd` then.

**Comment by xh3b4sd** [comment]
I commented here that we should have a TODO issue. Where is it? This is supposed to be tracked in the code with the TODO comment so we have a cross link and do not have to search.

**Pull request merged**